template.md

```diff
@@ -0,0 +1,305 @@
+
+## Basic Markdown, GFM, and special characters
+
+All basic and GitHub-flavored Markdown is supported. For more information on these, see:
+
+- [Baseline Markdown syntax](https://daringfireball.net/projects/markdown/syntax)
+- [GitHub-flavored Markdown (GFM) documentation](https://guides.github.com/features/mastering-markdown)
+
+Markdown uses special characters such as \*, \`, and \# for formatting. If you wish to include one of these characters in your content, you must do one of two things:
+
+- Put a backslash before the special character to "escape" it (for example, \\\* for a \*)
+- Use the [HTML entity code](http://www.ascii.cl/htmlcodes.htm) for the character (for example, \&\#42\; for a &#42;).
+
+## File name
+
+File names use the following rules:
+* Contain only lowercase letters, numbers, and hyphens.
+* No spaces or punctuation characters. Use the hyphens to separate words and numbers in the file name.
+* No more than 80 characters - this is a publishing system limit.
+* Use action verbs that are specific, such as develop, buy, build, troubleshoot. No -ing words.
+* No small words - don't include a, and, the, in, or, etc.
+* Must be in Markdown and use the .md file extension.
+
+## Headings
+
+Headings should be done using atx-style, that is, use 1-6 hash characters (#) at the start of the line to indicate a heading, corresponding to HTML headings levels H1 through H6. Examples of first- and second-level headers are used above. 
+
+There **must** be only one first-level heading (H1) in your topic, which will be displayed as the on-page title.  
+
+Second-level headings will generate the on-page TOC that appears in the "In this article" section underneath the on-page title.
+
+### Third-level heading
+#### Fourth-level heading
+##### Fifth level heading
+###### Sixth-level heading
+
+## Text styling
+
+*Italics* 
+
+**Bold** 
+
+~~Strikethrough~~
+
+## Links
+
+### Internal Links
+
+To link to a header in the same Markdown file, view the source of the published article, find the id of the head (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
+
+- Example: [Blockquotes](#blockquote)
+
+To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
+
+- Example: [Readme file](readme.md)
+- Example: [Welcome to .NET](./docs/welcome.md)
+
+To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
+
+- Example: [.NET Community](./docs/welcome.md#community)
+
+### External Links
+
+To link to an external file, use the full URL as the link.
+
+- Example: [GitHub](http://www.github.com)
+
+If a URL appears in a Markdown file, it will be transformed into a clickable link.
+
+- Example: http://www.github.com
+
+## Lists
+
+### Ordered lists
+
+1. This 
```

Comment:
I think it could be better to show the right numbers and then tell people that GitHub will put them in the right order even if they're not. but I'd like to confirm what other teams are doing here too. it does look odd to see a bunch of 1's. :smile: